internal/tiltfile/value/image.go

```diff
@@ -0,0 +1,46 @@
+package value
+
+import (
+	"fmt"
+
+	"github.com/docker/distribution/reference"
+	"go.starlark.net/starlark"
+)
+
+type ImageList []reference.Named
+
+var _ starlark.Unpacker = &ImageList{}
+
+// Unpack an argument that can be expressed as a list or tuple of image references.
+func (s *ImageList) Unpack(v starlark.Value) error {
+	*s = nil
+	if v == nil {
+		return nil
+	}
+
+	var iter starlark.Iterator
+	switch x := v.(type) {
+	case *starlark.List:
+		iter = x.Iterate()
+	case starlark.Tuple:
+		iter = x.Iterate()
+	default:
```

Comment:
IIRC if the arg is omitted, `v` will be `nil`, while if `image_deps=None` gets passed explicitly, it'll be `starlark.None`, so it'd be nice to treat that as an empty list here

Reply:
good catch! fixed